nvflare/app_common/workflows/cyclic_ctl.py

```diff
@@ -176,39 +176,47 @@ def _process_result(self, client_task: ClientTask, fl_ctx: FLContext):
         task = client_task.task
 
         result = client_task.result
+        client_name = client_task.client.name
         if isinstance(result, Shareable):
             # update the global learnable with the received result (shareable)
             # e.g. the received result could be weight_diffs, the learnable could be full weights.
             rc = result.get_return_code()
-            try:
```

Comment:
No, we should not change the order, because the last message may be early termination, but also contain the last training result, which should be converted to Learnable!